ci/Jenkinsfiles/build.groovy

```diff
@@ -29,21 +29,60 @@ String getCommitSha1() {
   return sh(returnStdout: true, script: 'git rev-parse HEAD').trim();
 }
 
+void dockerPull(String image) {
+  sh "docker pull ${image}"
+}
+
+void dockerTag(String image, String tag) {
+  sh "docker tag ${image} ${tag}"
+}
+
+void dockerPush(String image) {
+  sh "docker push ${image}"
+}
+
+void dockerDeploy(String imageName) {
+  String imageTag = "${ORG}/${imageName}:${VERSION}"
+  String internalImage = "${DOCKER_REGISTRY}/${imageTag}"
+  String explorerImage = "${DOCKER_REGISTRY}/${imageTag}"
+  echo "Push ${explorerImage}"
+  dockerPull(internalImage)
+  dockerTag(internalImage, explorerImage)
+  dockerPush(explorerImage)
+}
+
 pipeline {
   agent {
     label 'jenkins-nuxeo-package-11'
   }
   environment {
     APP_NAME = 'nuxeo-explorer'
+    DOCKER_REGISTRY = "?"
```

Comment:
I'd say `docker.packages.nuxeo.com` or`docker-private.packages.nuxeo.com` depenedning on the visibility.